apps/library/services/yandex_disk_export.py

```diff
@@ -0,0 +1,24 @@
+from pathlib import Path
+
+import yadisk
+from django.conf import settings
+
+
+def yandex_disk_export(instance):
+    yndx = yadisk.YaDisk(token=settings.YNDX_DISK_TOKEN)
+    cwd = str(settings.ROOT_DIR)
+    name = Path(str(instance.file)).name
+    year = str(Path(str(instance.file)).parent).split("\\")[-1]
+    to_dir = f"{year}/{name}"
+    from_dir = cwd.replace("\\", "/") + "/media/" + str(instance.file)
+    try:
+        yndx.mkdir(f"/{year}")
```

Comment:
Тут можно просто передать год, он правильно всё создаст, я приводил пример в пердыдущем ревью
```suggestion
        yndx.mkdir(year)
```